.github/workflows/ci.yml

```diff
@@ -14,10 +14,9 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-python@v4
-      with:
-        python-version: 3.7
-    - run: pip3 install --upgrade pip
+    - run: |
+        sudo apt-get install python3-venv make
+        pip3 install --upgrade pip
```

Comment:
a python3.12 miatt a venv-ekbe fel kell raknunk valoszinuleg a setuptools-ot is mert 3.12 ota nem rakja fel automatikusan a venv mint az elozo verziokban, ez valszeg eleg ha a requirements.in-be rakjuk,
persze ha nem hasznalunk olyan modult ami a setuptools-bol jon akkor targytalan

Reply:
Nem kéne semmit direktben használnunk a setuptoolsból.

Reply:
```
bldr/utils.py:4: in <module>
    from pkg_resources import resource_filename
E   ModuleNotFoundError: No module named 'pkg_resources'
```
ez a szitu python 3.12-vel, setuptools telepitese utan meg nincs ilyen gond a pkg_resources az a setuptools-bol jon

Reply:
De deprecated. Átírom inkább importlib_resources-ra, mint máshol is. Köszi, hogy jelezted!

Reply:
Ezzel az extra változtatással már működik 3.12-n?

Reply:
igen igy mar tuti